ttnn/cpp/ttnn/operations/pool/maxpool/device/kernels/compute/max_pool_multi_core.cpp

```diff
@@ -103,13 +97,19 @@ void MAIN {
     constexpr uint32_t num_faces_in_tile = is_partial_tile ? 1 : 2;
     constexpr uint32_t num_out_rows = 1;
 
-    constexpr uint32_t in_ntiles_hwc_block = in_ntiles_hwc / in_nblocks_c;
-    tilizeA_B_reduce_init<true>(in_cb_id, in_scalar_cb_id, in_ntiles_hwc_block, out_cb_id, num_faces_in_tile, window_size_hw);
-    pack_untilize_dst_init_short<num_output_tiles>(out_cb_id, num_out_rows, num_faces_in_tile); /* pack 1 row (1x16 or 1x32) */
+    constexpr uint32_t num_output_tiles = in_ntiles_c / in_nblocks_c;
```

Comment:
I think we need the total num tiles (including the height dim)?

Reply:
We just need the C tiles in this case I think because this kernel will never be called for large kernels, and thus in_ntiles_hw will always be 1.  I will add a comment.